content/doc/applications/docker/_index.md

```diff
@@ -33,6 +33,20 @@ Clever Cloud allows you to deploy any application running inside a Docker contai
 [FS Buckets](/doc/best-practices/cloud-storage/#what-is-fs-bucket) access, Dockerfile validation, and Docker Compose functionalities are not supported.
 {{< /callout >}}
 
+### How it works
+
+When you create a Docker application on Clever Cloud, the deployment process involves the following steps:
+
+1. **Install/Login:**
+   - The system checks for a Dockerfile and an entrypoint.
+   - It logs into the appropriate registry if necessary.
```

Comment:
```suggestion
   - It logs into a docker registry if configured
```

Reply:
I hesitated with this one because tagging Docker directly could be misleading. For example, Docker registry in GitHub is called Container registry.

Reply:
May I suggest being extra verbose?

> It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (note: the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)

(note: I'm not sure the docker registry is configure *inside the Dockerfile*)

Reply:
After internal discussion involving semantic king @Keksoj, this seems to be the best description:  
```suggestion
   -  It logs into the docker registry that was configured in the Dockerfile, if any, to find the necessary image (**note:** the name of the docker registry may vary depending on the provider. It is called "container registry" in Github, for instance)
```